apps/meteor/app/mentions/server/getMentionedTeamMembers.ts

```diff
@@ -1,35 +1,27 @@
 import { Team } from '@rocket.chat/core-services';
-import type { IMessage } from '@rocket.chat/core-typings';
 
 import { callbacks } from '../../../lib/callbacks';
 import { settings } from '../../settings/server';
 
-interface IExtraDataForNotification {
-	userMentions: any[];
-	otherMentions: any[];
-	message: IMessage;
+interface ITeamMention {
+	_id: string;
+	name: string;
+	type: string;
 }
 
-const beforeGetMentions = async (mentionIds: string[], extra?: IExtraDataForNotification) => {
-	const { otherMentions } = extra ?? {};
-
-	const teamIds = otherMentions?.filter(({ type }) => type === 'team').map(({ _id }) => _id);
-
-	if (!teamIds?.length) {
-		return mentionIds;
-	}
-
-	const members = await Team.getMembersByTeamIds(teamIds, { projection: { userId: 1 } });
-	mentionIds.push(...new Set(members.map(({ userId }) => userId).filter((userId) => !mentionIds.includes(userId))));
-
-	return mentionIds;
+const beforeGetTeamMentions = async (mentionIds: string[], teamMentions: ITeamMention[]): Promise<string[]> => {
```

Comment:
I think we can keep the method as it was, not sure why would you strip down the extra part